layouts/community/ergodox/pvazteixeira/README.md

```diff
@@ -0,0 +1,14 @@
+
+# pedro's ergodoxes
```

Comment:
```suggestion
# pedro's ergodoxes
```
Please also rename this file to lowercase.

Reply:
fixed in 225a260, 8cd7686. thanks